types/selenium-webdriver/lib/by.d.ts

```diff
@@ -105,6 +105,66 @@ export class By {
      */
     static xpath(xpath: string): By;
 
+    toObject(): Object;
+
+    /** @override */
+    toString(): string;
+}
+
+/**
+ * Describes a mechanism for locating an element relative to others
+ * on the page.
+ * @final
+ */
+export class RelativeBy {
```

Comment:
From reading [the docs](https://www.selenium.dev/documentation/webdriver/elements/locators/), it looks like a "relative locator" is a kind of locator. I think it should be added to the `Locator` type, so that you can pass it to e.g. `findElement`.